### ISSUE TITLE
DPI 565/666 support

### ISSUE THREAD

**Issue by kklem0** [pinout-xyz/Pinout.xyz#453]
I'm using a DPI 666 screen, would be nice to see what pins are left for other usages like i2c for touch screen.

gpio=0-9=a2
gpio=12-17=a2
gpio=20-25=a2

**Comment by lurch** [comment]
@Gadgetoid The "More Information" link on https://pinout.xyz/pinout/dpi needs updating to point to https://www.raspberrypi.com/documentation/computers/raspberry-pi.html#parallel-display-interface-dpi

**Comment by lurch** [comment]
@klem-everywhere As you can see in the table I linked to, there are several different ways of configuring the DPI 565 and 666 display modes.

**Comment by klem-everywhere** [comment]
> @klem-everywhere As you can see in the table I linked to, there are several different ways of configuring the DPI 565 and 666 display modes.

That's true, it would be very helpful if there's a way to show it so I could show others how I could connect a touch screen with different options. But it's true that's a very specific use case.
